core/domain/question_jobs_one_off.py

```diff
@@ -90,3 +90,24 @@ def reduce(key, values):
                 sum(ast.literal_eval(v) for v in values))])
         else:
             yield (key, values)
+
+
+class QuestionSummaryModelIndexesOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for indexing the creator_id for all
+    QuestionSummaryModels.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        """Return a list of datastore class references to map over."""
+        return [question_models.QuestionSummaryModel]
+
+    @staticmethod
+    def map(model_instance):
+        """Implements the map function for this job."""
+        model_instance.put(update_last_updated_time=False)
```

Comment:
Should we add a try-except block here to track models that fails update? (I'm not sure whether that can happen, @vojtechjelinek  do you have any thoughts?)

Reply:
In the few migrations, I did before we didn't have `try-except` blocks and everything went OK. If something magically fails it will be reported anyway.